DESCRIPTION

```diff
@@ -15,12 +15,13 @@ Description: Helps you imagine your data before you collect it. Hierarchical dat
 URL: https://declaredesign.org/r/fabricatr, https://github.com/DeclareDesign/fabricatr
 BugReports: https://github.com/DeclareDesign/fabricatr/issues
 Depends: R (>= 3.5.0)
-Imports: rlang (>= 0.2.0)
+Imports: rlang (>= 0.2.0), tibble
```

Comment:
Is tibble really required (not suggests)?

Reply:
Sorry missed this, will remove the dependency in the next maintenance release!